README.md

```diff
@@ -101,7 +101,7 @@ Audits/ Formal Verification
 ---------
 - [for Version 1.1.1 by G0 Group](docs/audit_1_1_1.md)
```

Comment:
Do you want to say `for Version 1.1.0 and 1.1.1`?

Reply:
do I? ... maybe ... wanted to focus in the versions that we used with our clients